src/auxiliary.rs

```diff
@@ -26,3 +30,18 @@ pub fn exec_user_command(args: &Option<String>) {
         None => error!("User command called without values"),
     }
 }
+
+pub fn atom_name(connection: &Arc<RustConnection>, id: u32) -> String {
+    let reply = connection.get_atom_name(id).unwrap().reply().unwrap();
+    connection.flush().unwrap();
```

Comment:
why is here a flush ?:)

Reply:
tested and was indeed not necessary -> is removed